### ISSUE TITLE
False positive 'Parentheses must be used when instantiating a new class'

### ISSUE THREAD

**Issue by Michiel-s** [squizlabs/PHP_CodeSniffer#3553]
**Describe the bug**
False positive phpcs error  'Parentheses must be used when instantiating a new class' for instantiating class with expanded namespace.

**Code sample**
```php
<?php

new \somenamespace\A();

```

**To reproduce**
1. Create a file called `test.php` with the code sample above...
2. Run `phpcs test.php --standard=PSR12`
3. See error message displayed
```
-------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-------------------------------------------------------------------------------------------
 3 | ERROR | [x] Parentheses must be used when instantiating a new class
-------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-------------------------------------------------------------------------------------------
```

**Expected behavior**
I expected this to be valid code.

**Versions (please complete the following information):**
 - OS: Ubuntu 20.04.3 LTS
 - PHP: PHP 8.1.2 (cli) (built: Jan 24 2022 10:42:33) (NTS)
 - PHPCS: PHP_CodeSniffer version 3.5.4 (stable) by Squiz (http://www.squiz.net)
 - Standard: PSR12

**Comment by Michiel-s** [comment]
Bug is somewhere introduced between v3.6.2 (no issue) en v3.5.4 (false positive)

**Comment by Michiel-s** [comment]
Sorry, duplicate of #3140 

Fixed already in later version.
